.gitignore

```diff
@@ -1,5 +1,11 @@
+# Backend stuff
+.idea/
+.vagga/
+runtime/data/*
 node_modules/
-log.txt
 config/config.js
+data.xml
+log.txt
+# Frontend stuff
 static/js/app.min.js
-static/css/style.min.css
+static/css/style.min.css
```

Comment:
Sure, I'll fix that everywhere. But genuine reason for the line endings is here:
http://stackoverflow.com/a/729795/2134936